cmd/get/clusters/command.go

```diff
@@ -0,0 +1,92 @@
+package clusters
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "clusters <cluster-id>"
+	alias = "cluster"
+
+	shortDescription = "Display one or many clusters"
+	longDescription  = `Display one or many clusters
+Output columns:
+- ID: Unique identifier of the cluster.
+- CREATED: Date and time of the Cluster CR creation.
```

Comment:
Suggest adding a few newlines here for breathability.

```suggestion

Output columns:

- ID: Unique identifier of the cluster.
- CREATED: Date and time of the Cluster CR creation.
```